apps/studio/components/interfaces/Home/ProjectList/ProjectList.tsx

```diff
@@ -37,7 +37,7 @@ const ProjectList = ({ rewriteHref }: ProjectListProps) => {
     isError: isErrorPermissions,
     error: permissionsError,
   } = usePermissionsQuery()
-  const { data: resourceWarnings } = useResourceWarningsQuery()
+  const { data: resourceWarnings } = useResourceWarningsQuery({ enabled: IS_PLATFORM })
```

Comment:
lets put this IS_PLATFORM check inside the react query instead? then its just one place instead of many